configs/RCCS-Fugaku-Fujitsu-A64FX-TofuD/auxiliary_software_files/packages.yaml

```diff
@@ -5,458 +5,206 @@
 
 packages:
   all:
-    compiler: [fj, gcc]
+    compiler: [fj, clang, gcc]
     providers:
       mpi: [fujitsu-mpi, openmpi, mpich]
       blas: [fujitsu-ssl2, openblas]
       lapack: [fujitsu-ssl2, openblas]
       scalapack: [fujitsu-ssl2, netlib-scalapack]
-      fftw-api: [fujitsu-fftw, fftw, rist-fftw]
+      fftw-api: [fujitsu-ssl2, fftw, rist-fftw]
```

Comment:
afaik, fujitsu-ssl2 provides fft functions, but for some reason the fftw.h header is missing. i'm also not yet 100% sure how to set it properly, when i compile (forgot) gromacs or hyper i see that it doesnt find the header and then builds fftw from source (which isn't the best approach, IMHO). i don't think 'fujitsu-fftw' exists, so the previous config is definitely wrong.

Reply:
What I mean is that because Spack's builtin `fujitsu-ssl2` package.py does not have `provides(fftw-api)` spack will never assume it is useful as an fftw provider and will always try to use something else (the logs generated by `ramble workspace setup` should include the concretization, and I assume Spack will have chosen something besides `fujitsu-ssl2` wherever an `fftw` implementation is needed).

Reply:
I see, thanks for the clarification